backend/src/test/java/de/sambam/gamenighttracker/service/GameServiceTest.java

```diff
@@ -23,4 +28,27 @@ public void addNewGameTester() {
         //THEN
         assertTrue(gameDb.getGameList().contains(Game.builder().name("Monopoly").build()));
     }
-}
\ No newline at end of file
+
+    @Test
+    @DisplayName("listAllGames should return all the games from db")
+    public void listAllGamesTest() {
+        //GIVEN
+        GameDb gameDb = new GameDb();
+        gameDb.add(Game.builder().name("CloudAge").releaseYear("2020").build());
```

Comment:
Better cut the db away and mock it here so you can test the service in isolation